src/test/java/com/yourcompany/SampleSauceTest.java

```diff
@@ -17,7 +19,7 @@
 import org.openqa.selenium.OutputType;
 
 
-@TestObject(testObjectApiKey = "INSERT_API_KEY_HERE", testObjectSuiteId = INSERT_SUITE_NUMBER_HERE)
+@TestObject(testObjectApiKey = "INSERT_API_KEY_HERE", testObjectSuiteId = 12345)
```

Comment:
@ragog I don't think we should hard-code the testId?
Let's remove that and use `INSERT_SUITE_NUMBER_HERE` or an environment variable?

Happy to discuss.